aports/main/postmarketos-mkinitfs/mkinitfs.sh

```
@@ -103,6 +103,24 @@ get_binaries()
 	lddtree -l $BINARIES | sort -u
 }
 
+# Collect non-binary files for osk-sdl and its dependencies
+get_osk_config()
+{
+	fontpath=$(awk '/^keyboard-font/{print $3}' /etc/osk.conf)
+	if [ ! -f $fontpath ]; then
+		echo "ERROR: Font specified in /etc/osk.conf does not exist!"
```

@craftyguy I didn't tested it but probably this error gets ignored becase `get_osk_config` is called from a subshell (`copy_files "$(get_osk_config)" "$tmpdir_extra"`)